app/views/devise/registrations/edit.html.haml

```diff
@@ -6,9 +6,17 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
     = devise_error_messages!
```

Comment:
Yes, for MA `create` we use this logic:
```
- if @membership_application.errors.any?
  - @membership_application.errors.full_messages.each do |msg|
    .wpcf7-response-output
      = msg
```
whereas in other places we use this kind of logic:
```
.wpcf7-response-output
  - @member_app_waiting_reason.errors.full_messages.each do |msg|
    = msg
```
The styling for `wpcf7-response-output` class includes `text-align:center;`.  If each message is separately styled (first example above) this forces each message to occur on a separate line.  The second example shows all messages on a single (centered) line.

We're all over the map here and thus we need to standardize this.

Also, here is another issue I just noticed: https://www.pivotaltracker.com/story/show/148990401